enterprise/messages.py

```diff
@@ -101,3 +101,26 @@ def add_unenrollable_item_message(request, item):
             span_end='</span>',
         )
     )
+
+
+def add_generic_info_message_for_error(request):
+    """
+    Add message to request indicating that there was an issue processing request.
+
+    Arguments:
+        request: The current request.
+
+    """
+    messages.info(
+        request,
+        _(
```

Comment:
Have you added the corresponding translations?

Reply:
Nope. @saleem-latif will add the translations in his [PR](https://github.com/edx/edx-enterprise/pull/217).